package.json

```diff
@@ -15,7 +15,7 @@
     "preintegration": "npm pack",
     "integration": "jest --config test/config/integration.json",
     "pretest": "npm run lint -- --cache",
-    "test": "jest",
+    "test": "cross-env LC_ALL=en-US jest",
```

Comment:
Thanks for this! I was hoping to get away from the brittle localized snapshots eventually, but this is a good addition regardless.